cmake/onnxruntime_mlas.cmake

```diff
@@ -458,9 +458,12 @@ else()
           list(APPEND ONNXRUNTIME_MLAS_LIBS onnxruntime_mlas_x86_64)
           set(mlas_platform_srcs )
         else()
-          set(MLAS_SOURCE_IS_NOT_SET 1)
+          set(MLAS_SOURCE_IS_NOT_SET 0)
         endif()
-
+    endif()
+    if(NOT ONNXRUNTIME_MLAS_MULTI_ARCH AND MLAS_SOURCE_IS_NOT_SET)
```

Comment:
Can it be reduced to 
> if (MLAS_SOURCE_IS_NOT_SET)
only?

Reply:
no, it can not. If ONNXRUNTIME_MLAS_MULTI_ARCH is set, MLAS_SOURCE_IS_NOT_SET will be true and actually we don't want a default build in that case.